### PR TITLE
update : responses.json & responses_th.json

### PR DESCRIPTION
**Pull request by Peachpiggies**
update & add : responses.json & responses_th.json

- update : responses.json : "another_world"

- add : "lazy" & "lazy_th" in both responses.json &  @responses_th.json
- add : "sleep" & "sleep_th" in both responses.json & responses_th.json

@StrixzIV # #

### CODE REVIEW

**Review by Peachpiggies — COMMENTED** [review]
I space out and accidentally press the mouse.

File: responses/responses.json [hunk 1]
```diff
@@ -176,6 +176,27 @@
...
     "is_single_response": false,
     "required_word": ["old"]
+  },
+
+  "lazy": {
+    "response": [
+      "Yeah me too. You should take a break.",
+      "Same to me. You should take a break."
+    ],
+    "list_of_words": ["I","am","so","lazy","exhausted","bored"],
```
Comment (StrixzIV) [inline responses/responses.json]: I think this intent word range is too wide. 

maybe it should be:
["i", "lazy", "exhausted", "bored"]

And also change all word in this list to lower case pls.

File: responses/responses.json [hunk 2]
```diff
@@ -176,6 +176,27 @@
...
+    "required_word": []
+  },
+
+  "sleep": {
+    "response": [
+      "You should hit the sack.",
+      "You should go to sleep.",
+      "You should take a nap."
+    ],
+    "list_of_words": ["I","am","so","sleepy"],
```
Comment (StrixzIV) [inline responses/responses.json]: This one is also too wide. I think it should be
["i", "sleepy"]

Change this one to all lower case too

File: responses/responses_th.json [hunk 3]
```diff
@@ -177,5 +177,27 @@
...
+    "list_of_words": ["เหนื่อย","เบื่อ","ขี้เกียจ"],
+    "is_single_response": false,
+    "required_word": []
+  },
+
+  "sleep_th": {
+    "response": [
+      "ไปนอนสิคะ คุณพี่ควรไปพักนะคะ",
+      "คุณพี่ควรไปนอนนะคะ",
+      "คุณพี่ควรไปนอนสิคะ"
```
Comment (StrixzIV) [inline responses/responses_th.json]: I think this should be: "คุณพี่ควรไปนอนได้แล้วนะคะ"

Comment (StrixzIV) [reply responses/responses_th.json]: And maybe add: "หนูว่าพี่ไปนอนเถอะค่ะ เดี๋ยวร่างกายจะไม่ไหวเอานะ"

**Review by StrixzIV — CHANGES_REQUESTED** [review]

File: responses/responses.json (hunk 2 above)
Comment (Peachpiggies) [reply responses/responses.json]: Ok

File: responses/responses_th.json (hunk 3 above)
Comment (Peachpiggies) [reply responses/responses_th.json]: Ok

File: responses/responses.json (hunk 1 above)
Comment (Peachpiggies) [reply responses/responses.json]: Ok

**Pull request merged**